### DISCUSSION TITLE
The latest GPT-4o model is available!

### DISCUSSION THREAD

**Discussion by keithschacht** [AllYourBot/hostedgpt#347] [General]
Just pull down main and install the app (or run migrations to upgrade). The app now uses the latest and greatest new GPT-4o model which outperforms everything else!
https://platform.openai.com/docs/models/gpt-4o

**Comment by VictorStastra** [comment]
Hey Keith, this doesn't seem to be working for me, 

I have the latest version forked and installed locally (just came across this repo today). I did a local install with docker compose and checked for migrations:
![image](https://github.com/allyourbot/hostedgpt/assets/30015175/a26e4799-086b-4096-b438-e47c76152674)

It seems that "Add new openai model" is included. 

However when I look at my latest requests in the usage dashboard of OpenAI I only see usage increasing for GPT4-Turbo: 
![image](https://github.com/allyourbot/hostedgpt/assets/30015175/4ddabb7a-e6f9-4acd-9d92-322c64b7404e)

**Reply by keithschacht** [reply]
@VictorStastra Oh shoot, I apologize! Indeed you are correct. I made sure to update all **existing** assistants to GPT-4o but I was not updating **newly created** assistants. I just updated `main` with a fix for this so go ahead and pull that down and re-run `bin/dev` (or simply re-do your deploy to Render or Fly) and everything should properly update this time. I appreciate you flagging this for me!
